src/ethereum_test_rpc/rpc.py

```diff
@@ -43,7 +47,28 @@ def __str__(self):
         if self.tx is not None:
             f"{super().__str__()} Transaction={self.tx.model_dump_json()}"
         elif self.tx_rlp is not None:
-            return f"{super().__str__()} Transaction RLP={self.tx_rlp.hex()}"
+            tx_rlp_hex_full = self.tx_rlp.hex()
+            # always log shortened errors to console (even when debugging read full from log file)
+            tx_rlp_hex = tx_rlp_hex_full[:50]
+
+            # create ./logs/rlp folder if it does not exist already
+            rlp_logs_folder = Path(".") / "logs" / "rlp"
```

Comment:
We have `AppConfig().DEFAULT_EVM_LOGS_DIR` instead of `Path(".")` but I'm not sure if that's the correct source for this location, so I'll defer to @danceratopz who is more familiar with the consume command logging.